components/studio/bin/hab-studio.sh

```diff
@@ -1011,6 +1016,9 @@ report_env_vars() {
   if [ -n "${HAB_ORIGIN:-}" ]; then
     info "Exported: HAB_ORIGIN=$HAB_ORIGIN"
   fi
+  if [ -n "${HAB_AUTH_TOKEN:-}" ]; then
+    info "Exported: HAB_AUTH_TOKEN=$HAB_AUTH_TOKEN"
```

Comment:
Do we think it could be worth redacting this value from the logs?  My concern would be users who are sharing build logs with each other to help solve problems, not realizing they are revealing their auth token.

Reply:
This could be a case where we could display: `Exported: HAB_AUTH_TOKEN=<redacted>` to make it clear that a value cross the Studio barrier but that its value is sensitive.